build.gradle

```diff
@@ -1,29 +1,29 @@
-group 'io.github.io.github.sskorol'
-version '0.0.1'
+group 'gradle project'
+version '1.0.1'
 
 apply plugin: 'java'
 
 sourceCompatibility = 1.8
 
+project.ext {
+    mainClassName = 'com.company.PreviousProject.Main'
+}
+
 repositories {
-    jcenter()
+    mavenCentral()
```

Comment:
@lmalakhova I think it's better to use jcenter as it have more relevant versions of libs.